src/status_im2/contexts/onboarding/generating_keys/view.cljs

```diff
@@ -125,7 +125,8 @@
      :container-style   {:top  40
                          :left 20}
      :layers            (:generate-keys resources/parallax-video)
-     :disable-parallax? true}]
+     :disable-parallax? true
+     :enable-looping?   false}]
```

Comment:
We need to default is to true, Because in every other implementation that we have for the transparent video it is looping.

Reply:
Yes but in the sake of a (in my opinion) coherent api I would say this is worth it.
Additionally we can just change the name of the prop so that it suits the use case here since we are also creating the package. 
It's a minor detail but imo details like this add value. 
Cc @briansztamfater

Reply:
Just realized, It'll be more confusing xD
```clj 
:loop   (not disable-looping?)
```

Reply:
ah got it. cool, thanks @ibrkhalil!

Reply:
I mean, we can just enable looping on the 2 uses where it is used too

Reply:
mostly my concern was with the package, so whatever you go with is fine. I see now that the real api is `loop` 👍

Reply:
Created a PR [here](https://github.com/status-im/react-native-transparent-video/pull/5) to rename it

Reply:
I think I'll make that PR a draft for now, And we can continue this task